build/build.fsproj

```diff
@@ -18,15 +18,15 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="BlackFox.Fake.BuildTask" Version="0.1.3" />
-    <PackageReference Include="Fake.Api.Github" Version="5.22.0" />
-    <PackageReference Include="Fake.Core.Process" Version="5.22.0" />
-    <PackageReference Include="Fake.Core.ReleaseNotes" Version="5.22.0" />
-    <PackageReference Include="Fake.Core.Target" Version="5.22.0" />
-    <PackageReference Include="Fake.DotNet.Cli" Version="5.22.0" />
-    <PackageReference Include="Fake.DotNet.MSBuild" Version="5.22.0" />
-    <PackageReference Include="Fake.IO.FileSystem" Version="5.22.0" />
-    <PackageReference Include="Fake.Tools.Git" Version="5.22.0" />
+      <PackageReference Include="BlackFox.Fake.BuildTask" Version="0.1.3" />
+      <PackageReference Include="Fake.Api.Github" Version="6.0.0" />
+      <PackageReference Include="Fake.Core.Process" Version="6.0.0" />
+      <PackageReference Include="Fake.Core.ReleaseNotes" Version="6.0.0" />
+      <PackageReference Include="Fake.Core.Target" Version="6.0.0" />
```

Comment:
Just a quick question: Wasn't it the case that we wanted to keep all version tags exact? (I.e., `Version="[6.0.0]"`)

Reply:
I think in the case of the build project it does not matter, but feel free to pin